eng/Diags/Microsoft.Analyzers.Extra.yml

```diff
@@ -1,256 +1,176 @@
 Origin:
-  AssemblyName: Microsoft.Extensions.ExtraAnalyzers.Roslyn4.0
+  AssemblyName: Microsoft.Analyzers.Extra
   Version: 42.42.42.42424
 Diagnostics:
-  R9A000:
+  EA0012:
     Metadata:
       Category: Performance
-      Title: Use source generated logging methods for improved performance
-      Description: Identifies calls to legacy logging methods
-      HelpLinkUri: https://TODO/r9a000
```

Comment:
Why have we removed the links? We need to have a doc (e.g., similar to https://github.com/dotnet/extensions/blob/main/docs/list-of-diagnostics.md) that contains the list of diagnostics and associated docs. 
For the ease of maintenance, the doc should probably be generatable too.